src/Qir/Runtime/lib/Simulators/ToffoliSimulator.cpp

```diff
@@ -109,11 +109,22 @@ namespace Quantum
             return std::abs(actualZeroProbability - probabilityOfZero) < precision;
         }
 
+        // To do: remove after the `function DumpMachine<'T> (location : 'T) : Unit` is implemented.

```

Comment:
As before, I'm not sure that I agree with removing this; it's still useful to host programs.
```suggestion
        // TODO: remove after the `function DumpMachine<'T> (location : 'T) : Unit` is implemented.

```

Reply:
Fixed.